src/types/fast-deep-equal.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'fast-deep-equal';
```

Comment:
But `fast-deep-equal` has typings out of the box based on NPM
<img width="582" height="177" alt="image" src="https://github.com/user-attachments/assets/d0495dad-0435-4095-9a0d-b9b4e81a6bf7" />